### PR TITLE
Passing unspecified Coppa when publisher is not setting it explicitly. 

### PR DESCRIPTION
**Pull request by prabodhprakash**

### CODE REVIEW

**Comment by prabodhprakash** [comment]
@googlebot I fixed it.

**Comment by prabodhprakash** [comment]
@googlebot I consent.

**Comment by itiagrawal1** [comment]
@googlebot I consent.

**Comment by itiagrawal1** [comment]
@googlebot I fixed it.

File: adapters/InMobi/InMobiAdapter/GADMAdapterInMobi.m [hunk 1]
```diff
@@ -128,10 +128,9 @@ - (void)prepareRequestParameters {
   GADMAdapterInMobiMutableDictionarySetObjectForKey(dict, @"tp", @"c_admob");
   GADMAdapterInMobiMutableDictionarySetObjectForKey(dict, @"tp-ver", [GADRequest sdkVersion]);
 
-  if ([[self.connector childDirectedTreatment] integerValue] == 1) {
-    GADMAdapterInMobiMutableDictionarySetObjectForKey(dict, @"coppa", @"1");
-  } else {
-    GADMAdapterInMobiMutableDictionarySetObjectForKey(dict, @"coppa", @"0");
+  if ([self.connector childDirectedTreatment]) {
+    NSString *coppaString = [[self.connector childDirectedTreatment] integerValue]? @"1" : @"0";
+    GADMAdapterInMobiMutableDictionarySetObjectForKey(extrasDictionary, @"coppa", coppaString);
```
Comment (chlandacastro) [inline adapters/InMobi/InMobiAdapter/GADMAdapterInMobi.m:133]: compiling error: Use of undeclared identifier 'extrasDictionary'

Comment (itiagrawal1) [reply adapters/InMobi/InMobiAdapter/GADMAdapterInMobi.m]: Changes done.

**Pull request merged**